File: Barbearia/app/src/main/java/com/example/barbearia/MainActivity.java
package com.example.barbearia;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import com.example.barbearia.telas.CadastroActivity;

public class MainActivity extends AppCompatActivity {

    private EditText email;
    private  EditText senha;
    private TextView resultado;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        email = findViewById(R.id.email);
        senha = findViewById(R.id.senha);
        resultado = findViewById(R.id.resultado);
    }

    public void irTelaCadastro(View view){
        Intent intent = new Intent(MainActivity.this, CadastroActivity.class);
        startActivity(intent);

    }
    public void Logar(View view){
        if(senha.getText().toString().isEmpty() || email.getText().toString().isEmpty()){

            resultado.setText("Por favor preenchar os campos senha e email corretamentes");

        }else{

        resultado.setText("sistema incompleto bd caro cliente:" + email.getText().toString());

        }

    }
}

File: MyKotlinaplication/app/src/main/java/com/example/mykotlinaplication/MainActivity.kt
package com.example.mykotlinaplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
val senhaPad: String ="12345"
val emailPad: String ="marcos"
var buton: Button? = null;
var email: EditText? = null
var senha: EditText? = null
var resultado:TextView? =null

class MainActivity : AppCompatActivity(),View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

         buton = this.findViewById(R.id.logar)
         email = this.findViewById(R.id.email)
         senha = this.findViewById(R.id.senha)
         resultado=this.findViewById(R.id.resultado);




    }



    override fun onClick(v: View?) {

        this.validar()

    }


    public fun irMenu(){
        intent =  Intent(this,MainActivity2::class.java)
        startActivity(intent)
    }

    public  fun validar() {
        if (email!!.text.isEmpty()|| senha!!.text.isEmpty()){
            resultado!!.setText("senha ou login invalidos")

        }else if (senhaPad != senha!!.text.toString() && emailPad==email!!.text.toString()){
            resultado!!.setText("senha não existe")
        }else if(senhaPad != senha!!.text.toString() && emailPad!=email!!.text.toString()){
            resultado!!.setText("email e senha não batem")
        }
        else{
            this.irMenu()
        }
    }

}